package org.pemrogramanlanjut.tugas.ExceptionsListsThreadsandFiles;

public class module {
    void myFunc(int a, int b) {

        try {

            System.out.println(a / b);
         }

         catch (Exception e) {

             System.out.println("Error");
         }
    }
}

